prow/ANNOUNCEMENTS.md

```diff
@@ -183,6 +183,16 @@ state and no claims of backwards compatibility are made for any external API.
 Note: versions specified in these announcements may not include bug fixes made
 in more recent versions so it is recommended that the most recent versions are
 used when updating deployments.
+ - *February 16th, 2022* Since prow version `v20220217-362d6ac4a0`, the
+   entrypoint container in a prow job will run as `--copy-mode-only`, instead of
+   `/bin/cp /entrypoint /tools/entrypoint`. Entrypoint images before the mentioned version
+  will not work with `--copy-mode-only`, and entrypoint image since the
+  mentioned version will not work with `/bin/cp /entrypoint /tools/entrypoint`.
```

Comment:
> and entrypoint image since the  mentioned version will not work with `/bin/cp /entrypoint /tools/entrypoint`.

Really, why?

Reply:
because in the image produced by `ko` the binary is stored under `/ko-app/` instead of root level